site/docs/types.md

```diff
@@ -73,6 +84,12 @@ Accepts number values. For example: `1`, `3.14`, `1e5`.
 
 Accepts object literals. For example: `{"left":5, "right":30, "top":5, "bottom":50}`. The valid object property names and types will vary across parameters; read the individual parameter descriptions for more information.
 
+<br/><a name="overlap" href="#overlap">#</a>
+**Overlap**
```

Comment:
This is only for axis and isn't used elsewhere. Better make the type for it clear in the table and don't require jumping to the type page.

Reply:
Done

Reply:
That's not true. Legends support it as well.

Reply:
I removed these anyway.